pkg/operator/csi/csidrivercontrollerservicecontroller/csi_driver_controller_service_controller.go

```diff
@@ -29,6 +30,8 @@ const (
 	resizerImageEnvName       = "RESIZER_IMAGE"
 	snapshotterImageEnvName   = "SNAPSHOTTER_IMAGE"
 	livenessProbeImageEnvName = "LIVENESS_PROBE_IMAGE"
+
+	infraConfigName = "cluster"
```

Comment:
over-abstraction

Reply:
Sorry, do you mind elaborating?

Not sure if you're talking about this, but I've seen this pattern (const with singleton CR name) used  across most controllers in library-go.